tests/test_solvers_fairflow.py

```diff
@@ -185,7 +185,7 @@ def test_solver_fairflow_no_0_score_assignment():
     Tests 5 papers, 4 reviewers.
     Reviewers review min: 1, max: 3 papers.
     Each paper needs 2 reviews.
-    No constraints.
+    Reviewer 0 cannot review paper 1.
```

Comment:
The constraint matrix inside the test has a -1 in element [0, 1], so there actually is this constraint.